src/util/ld_util.py

```diff
@@ -181,6 +205,38 @@ def get_triples_for_blank_node_from_rdf_store(
     return g
 
 
+def get_label_triples_and_types_for_entities_and_roles_from_rdf_store(
+    resource_url: str, sparql_endpoint: str
+) -> Graph:
+    """Returns a graph with label triples for the labels of entities
+     and roles attached to the LOD resource
+    (including preflabels for the SKOS Concepts from the rdf store).
+    """
+    # get the preflabels and types for the entities...
+    query_construct_person_pref_labels = (
+        f"PREFIX skosxl: <http://www.w3.org/2008/05/skos-xl#> "
+        f"CONSTRUCT {{ ?z skos:prefLabel ?pref_label . ?z rdf:type ?t }} WHERE {{ "
+        f"VALUES ?s {{ <{resource_url}> }} . ?s ((sdo:creator/sdo:creator)|(sdo:byArtist/sdo:byArtist)|(sdo:actor/sdo:actor)|(sdo:contributor/sdo:contributor)|(sdo:mentions/sdo:mentions)|(sdo:productionCompany/sdo:productionCompany)) ?z . ?z skosxl:prefLabel/skosxl:literalForm ?pref_label .?z rdf:type ?t }}"
+    )
+    g = sparql_construct_query(sparql_endpoint, query_construct_person_pref_labels)
+
+    # get the additional types for the entities...
+    query_construct_person_pref_labels = (
+        f"PREFIX skosxl: <http://www.w3.org/2008/05/skos-xl#> "
+        f"CONSTRUCT {{ ?z skos:prefLabel ?pref_label . ?z sdo:additionalType ?t}} WHERE {{ "
+        f"VALUES ?s {{ <{resource_url}> }} . ?s ((sdo:creator/sdo:creator)|(sdo:byArtist/sdo:byArtist)|(sdo:actor/sdo:actor)|(sdo:contributor/sdo:contributor|(sdo:mentions/sdo:mentions)|(sdo:productionCompany/sdo:productionCompany))) ?z . ?z skosxl:prefLabel/skosxl:literalForm ?pref_label .?z sdo:additionalType ?t}}"
+    )
```

Comment:
op één of andere manier wordt sdo:Person toegevoegd aan een Concept dat het "Metropole Orkest" voorstelt. 
Zie bijvoorbeeld:
`curl -X 'GET' \
  'http://127.0.0.1:5000/id/program/2101712250239865931' \
  -H 'accept: text/turtle'`

Reply:
net even gecheckt op productie: ook daar is er een sdo:Person
deze PR zorgt er voor dat er ook de overige types bij worden gezet plus een label. 
lijkt me wel een verbetering